### PR TITLE
Bugzilla doc text for 4.5.2 release

### PR DESCRIPTION
**Pull request by jeana-redhat**
- Updated 4.5 release info to mention 4.5.2 and point to corresponding advisory
- Removed 4.5.1 Known Issue BZ1856501
- Made changes to 4.5.1 text to indicate 4.5.2 is the GA release
- Added 4.5.2 RN text and links

### CODE REVIEW

**Comment by jeana-redhat** [comment]
Eric, Vikram, and Cody - I sent you a note about my availability this evening via email (in case you see this first).

**Review by codyhoag — APPROVED** [review]
In terms of required updates and general things to call out, LGTM! I'll defer to Vikram/Eric for customer messaging tweaks. Thanks!

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -21,12 +21,12 @@ compliance, and governance requirements.
...
 
 Red Hat {product-title}
-(link:https://access.redhat.com/errata/RHBA-2020:2409[RHBA-2020:2409]) is now
+(link:https://access.redhat.com/errata/RHBA-2020:2909[RHBA-2020:2909]) is now
 available. This release uses link:https://v1-18.docs.kubernetes.io/docs/setup/release/notes/[Kubernetes 1.18] with CRI-O runtime. New features, changes, and known issues that pertain to
 {product-title} {product-version} are included in this topic.
 
 Red Hat did not publicly release {product-title} 4.5.0 as the GA version and,
-instead, is releasing {product-title} 4.5.1 as the GA version.
+instead, is releasing {product-title} 4.5.2 as the GA version.
```
Comment (sferich888) [inline release_notes/ocp-4-5-release-notes.adoc:29]: No! We GA'ed 4.5.1 (it was the version we first shipped for 4.5). There is no need for this. 
What version we announce with; is irrelevant, compared to what version we GA (actually ship with). 

cc: @reestr

File: release_notes/ocp-4-5-release-notes.adoc [hunk 2]
```diff
@@ -1971,8 +1964,10 @@ properly.
 
 Issued: 2020-07-13
 
-{product-title} release 4.5 is now available. The list of container images and
-bug fixes includes in the update are documented in the
+Red Hat did not publicly release {product-title} 4.5.0 or 4.5.1 as the GA
```
Comment (sferich888) [inline release_notes/ocp-4-5-release-notes.adoc:1967]: But we did ship 4.5.1, so I don't think this is accurate.

File: release_notes/ocp-4-5-release-notes.adoc (hunk 1 above)
Comment (vikram-redhat) [reply release_notes/ocp-4-5-release-notes.adoc:29]: @sferich888 if we don't mention this, aren't we likely to get questions about why customers can't upgrade to 4.5.1 and instead are being upgraded to 4.5.2 directly?

Comment (reestr) [reply release_notes/ocp-4-5-release-notes.adoc:29]: 4.5.1 is the release for 4.5 GA.  We can't simply change that now without causing more confusion for customers. Some have already installed and getting support.  Let's please ensure to have consistent messaging. Only delay is with the marketing announcement and that is instead with 4.5.2.

File: release_notes/ocp-4-5-release-notes.adoc (hunk 2 above)
Comment (codyhoag) [reply release_notes/ocp-4-5-release-notes.adoc:1967]: @sferich888 Could we instead just state that it's not available for upgrades? And leave the rest as-is?:
```
Upgrades to version 4.5.1 are blocked; the 4.5.1 advisories are available for reference only.
```

**Comment by sferich888** [comment]
Were getting them now, we get them for every release, we don't ship a
stable upgrade point until the first z steam anyways.

File: release_notes/ocp-4-5-release-notes.adoc (hunk 1 above)
Comment (vikram-redhat) [reply release_notes/ocp-4-5-release-notes.adoc:29]: @reestr how can we have a different marketing announcement to the docs? Both docs and marketing need to have the same message?

If the marketing announcement says 4.5.2, and docs say 4.5.1, I can expect bugs coming, not to mention confusion in the field.

Comment (jeana-redhat) [reply release_notes/ocp-4-5-release-notes.adoc:29]: Does something like this thread the needle?
`Red Hat did not publicly release {product-title} 4.5.0 as the GA version and, instead, released {product-title} 4.5.1 as the GA version. However, due to an upgrade issue, 4.5.1 has been pulled. A new upgrade path from 4.4.z to 4.5.1 and 4.5.1 to 4.5.2 will be available with the 4.5.2 release.`

Comment (nstielau) [reply release_notes/ocp-4-5-release-notes.adoc:29]: That seems OK to me.  Technically, we didn't pull anything, we just didn't publish a blog post.  I'd rather just say 4.5.1 was GA and change marketing language to say that 4.5 is GA and point to 4.5.2 as the most recent release.

Comment (jeana-redhat) [reply release_notes/ocp-4-5-release-notes.adoc]: Updated with slight tweak:
```
Red Hat did not publicly release {product-title} 4.5.0 as the GA version and,
instead, released {product-title} 4.5.1 as the GA version. However, 4.5.1
has an upgrade issue (link:https://bugzilla.redhat.com/show_bug.cgi?id=1856501[*BZ#1856501*]).
A new upgrade path from 4.4.z to 4.5.1 and 4.5.1 to 4.5.2 is available with the
4.5.2 release.
```

**Comment by jeana-redhat** [comment]
Sorry for force commit notification churn - today I learned more about Git! :smiley: Anyway, this should cover the discussion over the last 24 hours, PTAL :pray:

**Comment by codyhoag** [comment]
Updated text now follows general z-stream format and language. LGTM :heavy_check_mark:

**Pull request merged**